src/plugins/intel_gpu/src/graph/common_utils/common_tools.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2025 Intel Corporation
```

Comment:
Do we need this?
We already have similar helpers in src/plugins/intel_gpu/include/intel_gpu/runtime/utils.hpp

Reply:
No, with this file there is no need anymore, thanks, I didn't see this